### ISSUE TITLE
Options missing in config.json

### ISSUE THREAD

**Issue by WSSDude420** [yamashi/PerformanceOverhaulCyberpunk#160]
I found out that with latest version, few more things were added, but they are not inside packaged config.json.

More specifically, these settings are missing:
```
    "remove_pedestrians": false,
    "skip_start_menu": true,
    "disable_async_compute": false,
    "disable_antialiasing": false,
    "disable_intro_movies": true,
    "disable_vignette": false,
    "disable_boundary_teleport": false
```
(NOTE: values are mine, don't know defaults...)
 
 They get added there automatically (as they should) after first launch... But still, would be better to have them in as-is I believe :)
 
 Also, `unlock_menu` seems to be renamed to `enable_debug` now, so probably should change that as well inside config.json.

**Comment by hulovexu1314** [comment]
Brother will pay attention!

**Comment by yamashi** [comment]
Thanks fixed in 1.3.0
